autopush-common/src/db/models.rs

```diff
@@ -70,6 +71,8 @@ pub struct DynamoDbUser {
     pub uaid: Uuid,
     // Time in milliseconds that the user last connected at
     pub connected_at: u64,
+    // When a user is considered abandoned (updated frequently)
+    pub expiry: u64,
```

Comment:
IIRC this will trigger `serde_dynamodb` deserialization errors on user records that lack an `expiry` field which we definitely have. If that's the case this needs to be `Option<u64>`.